### DISCUSSION TITLE
How can i disable the call self as `node` binary?

### DISCUSSION THREAD

**Discussion by viceice** [yao-pkg/pkg#25] [Q&A] [answered]
I'm building some tooling which needs to call the native system node. but intead pkg is trying to execute the current binary as node.

Is there any hint where to patch so this is disabled? I would also like to disable `PKG_EXECPATH` on child env as i also need to call the binary again with other parameters.

So my tool is some kind of a multicall binary.

- https://github.com/containerbase/base

**Comment by viceice** [comment]
I run commands like `containerbase-cli df ...` or `containerbase-cli it`.
if the binary is symlinked to `install-tool` then it will internally run `containerbase-cli it`.

**Comment by robertsLando** [comment]
Try checking `prelude/bootstrap`, if not there it may be something related to nodejs patches

**Reply by viceice** [reply]
i see. so it's maybe simply enough to disable the `child_process` patching?

**Reply by robertsLando** [reply, marked as answer]
It seems so: https://github.com/yao-pkg/pkg/blob/main/prelude/bootstrap.js#L2059

**Reply by viceice** [reply]
seems to work, thanks

- https://github.com/containerbase/base/pull/2137

**Reply by robertsLando** [reply]
you welcome
